packages/cfpb-atomic-component/src/utilities/dom-traverse.js

```diff
@@ -15,34 +15,47 @@ function queryOne( expr, con ) {
 /**
  * Get the nearest parent node of an element.
  *
- * @param {HTMLNode} element - A DOM element.
+ * @param {HTMLNode} elem - A DOM element.
  * @param {string} selector - CSS selector.
  * @returns {HTMLNode} Nearest parent node that matches the selector.
  */
-function closest( element, selector ) {
-  if ( 'closest' in element ) {
-    return element.closest( selector );
-  }
+function closest( elem, selector ) {
+  elem = elem.parentNode;
```

Comment:
This line is causing the tests to fail. We immediately override elem with its parent which makes sense because we're looking for the "nearest parent node of an element" but it's a departure from the current functionality.